src/components/Dropdown.tsx

```diff
@@ -1,19 +1,31 @@
-import { useState, useEffect } from "react";
+import {
+  useState,
+  useEffect,
+  ReactElement,
+  FocusEvent,
+  MouseEvent,
+} from "react";
+import { OrderBy } from "@/apis/apis.type";
 import "./Dropdown.css";
 
-export default function Dropdown({ onSelect, children }) {
-  const [isActive, setIsActive] = useState(false);
-  const [value, setValue] = useState(children[0].props.value);
+interface Props {
```

Comment:
P2:
이름은 더 명확한 것이 좋습니다. 나중에 이런 type이 export 될 가능성이 있기 때문에 더 자세하게 적어주시는 것이 필요합니다.
```suggestion
interface DropdownProps {
```